db_stress_tool/expected_state.cc

```diff
@@ -273,7 +273,7 @@ Status FileExpectedStateManager::SaveAtAndAfter(DB* db) {
     s = FileSystem::Default()->RenameFile(state_file_temp_path, state_file_path,
                                           IOOptions(), nullptr /* dbg */);
   }
-  SequenceNumber old_saved_seqno;
+  SequenceNumber old_saved_seqno = 0;
```

Comment:
Note to reviewer:
I found it surprising that there is zero-initializing of SequenceNumber scattering across the codebase but not any noticable description of a "normal" SequenceNumber is greater than 0. Is [`const SequenceNumber kMinUnCommittedSeq = 1;  // 0 is always committed` in `types.h`](https://github.com/facebook/rocksdb/blob/main/include/rocksdb/types.h#L20) for that?

Reply:
I would've preferred to leave it uninitialized rather than pick a value at the declaration site that we expect to be overwritten before it's used. But gcc with flags like `-Wmaybe-uninitialized -Werror -fsanitize=address` doesn't seem to realize it's assigned prior to use so we have to pick some initial value. Any value should work but something like zero is preferable because it's easy for the reader to ignore (unlike if we chose 3 or something, for example).